go/oasis-test-runner/oasis/fixture.go

```diff
@@ -393,6 +396,8 @@ type ComputeWorkerFixture struct {
 
 	// Runtimes contains the indexes of the runtimes to enable.
 	Runtimes []int `json:"runtimes,omitempty"`
+	// RuntimeStatePaths are the paths to runtime state that will be copied to the compute worker node.
+	RuntimeStatePaths []string `json:"runtime_state_paths,omitempty"`
```

Comment:
We should instead change the `RuntimeStatePaths` to be a map, similar to the `RuntimeConfig`, e.g. `map[int]string`. This would avoid the runtime alignment requirement.